### PR TITLE
add ProxyErrorOverride directive

### PR DESCRIPTION
**Pull request by timogoebel**
This PR adds support for the ProxyErrorOverride directive. http://httpd.apache.org/docs/2.2/mod/mod_proxy.html#proxyerroroverride

### CODE REVIEW

**Comment by mhaskel** [comment]
@timogoebel Thanks for the contribution! Could you update the [vhost unit tests](https://github.com/puppetlabs/puppetlabs-apache/blob/master/spec/defines/vhost_spec.rb) to exercise this new parameter?

**Comment by timogoebel** [comment]
@mhaskel: I added the same tests as for proxy_preserve_host. What do you think?

File: spec/defines/vhost_spec.rb [hunk 1]
```diff
@@ -192,6 +192,7 @@
           'php_admin_values'            => ['true', 'false'],
           'no_proxy_uris'               => '/foo',
           'proxy_preserve_host'         => true,
+          'proxy_error_override'        => true,
```
Comment (igalic) [inline spec/defines/vhost_spec.rb]: you're not checking this anywhere ;)

Comment (timogoebel) [inline spec/defines/vhost_spec.rb]: Well yes, however proxy_preserve_host is not checked anywhere as well, is it?

Comment (igalic) [inline spec/defines/vhost_spec.rb]: [inconsolable sobbing]

Comment (timogoebel) [inline spec/defines/vhost_spec.rb]: I was unter the impression, that the context was just to "set everything!" and check whether the manifest compiles. What tests do you think are necessary?

**Comment by timogoebel** [comment]
@igalic @mhaskel : Any more feedback on this? Are the tests ok? Can this be merged?

**Comment by igalic** [comment]
@timogoebel i'll merge this one, and i've created an issue to track our broken tests: [MODUlES-1774](https://tickets.puppetlabs.com/browse/MODULES-1774)

thank you for this patch!

**Pull request merged**